app/policies/supporting_document_policy.rb

```diff
@@ -0,0 +1,9 @@
+class SupportingDocumentPolicy < ApplicationPolicy
+  def download?
+    if user.review_manager? || user.reviewer?
+      record.permit_application.jurisdiction.id == user.jurisdiction_id
+    elsif user.submitter?
+      record.permit_application.submitter_id == user.id
```

Comment:
what about super_admin?

Reply:
They cannot see this. super admin has no access